### PR TITLE
Release 4.0.4

### PR DESCRIPTION
**Pull request by FinalAngel**
Fixes issue:

image

### CODE REVIEW

**Comment by Brankub** [comment]
LGTM

File: templates/includes/menu/langnav.html [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-            {% endif %}
-        {% endfor %}
-    </ul>
-</li>
+    <li class="child lang dropdown">
+        <a href="{% page_language_url lang.code %}" class="dropdown-toggle" data-toggle="dropdown"><span>{{ lang.name }} <span class="caret"></span></span></a>
+        <ul class="dropdown-menu" role="menu">
+            {% for language in languages %}
+                {% if not language.0 == current_language %}
+                    <li class="child lang"><a href="{% page_language_url language.0 %}"><span>{{ language.1 }}</span></a></li>
```
Comment (vxsx) [inline templates/includes/menu/langnav.html]: can we also fix it so language name is rendered in that language and not in the user's current language while we are at it?

Comment (vxsx) [inline templates/includes/menu/langnav.html]: e.g. now its
*en*
```
English
German
```
*de*
```
Englisch
Deutsch
```

but should be

```
English
Deutsch
```
always

Comment (FinalAngel) [inline templates/includes/menu/langnav.html]: @vxsx can you create a ticket for this fix? This PR meant for a critical issue where language isn't even displayed.

**Comment by vxsx** [comment]
:+1:

**Pull request merged**